Frontend/ui-driver/src/Screens/DriverProfileScreen/View.purs

```diff
@@ -1259,9 +1261,9 @@ infoTileView state config =
                     ][imageView
                         [ height $ V 13
                         , width $ V 13
-                        , imageWithFallback if item <= (fromMaybe 0.0 state.data.analyticsData.rating) then "ny_ic_star_active,https://assets.juspay.in/nammayatri/images/common/ny_ic_star_active.png" else "ny_ic_star_inactive,https://assets.juspay.in/nammayatri/images/common/ny_ic_star_inactive.png"
+                        , imageWithFallback (getStarsForRating item (fromMaybe 0.0 state.data.analyticsData.rating))
```

Comment:
@dhruvSHA256 this function will be called for each iteration instead why don't we call this function only once and pass as a argument to `map` and if its `1.0` show full star else half star